imageio/plugins/ffmpeg.py

```diff
@@ -534,13 +614,41 @@ def _initialize(self):
                    '-pix_fmt', self._pix_fmt,
                    '-r', "%.02f" % fps,
                    '-i', '-', '-an',
-                   '-vcodec', codec] 
-            cmd += ['-b', str(bitrate)] if (bitrate is not None) else [] 
-            cmd += ['-r', "%d" % fps, self._filename]
-            
+                   '-vcodec', codec,
+                   '-pix_fmt', pixelformat,
+                   ]
+            if bitrate is not None:
+                cmd += ['-b:v', str(bitrate)]
+            if quality >= 0:  # If < 0, then we don't add anything
```

Comment:
@kuchi I like the quality argument. But I would ditch the -1 option, and simply use bitrate instead if its given. In other words, make this an `else`.